src/github.com/matrix-org/dendrite/common/config/config.go

```diff
@@ -360,6 +371,14 @@ func (config *Dendrite) derive() {
 		config.Derived.Registration.Flows = append(config.Derived.Registration.Flows,
 			authtypes.Flow{Stages: []authtypes.LoginType{authtypes.LoginTypeDummy}})
 	}
+
+	// Load application service configuration files
+	fmt.Println(config.ApplicationService.ConfigFiles)
```

Comment:
This should be a `logrus` thing, no? Or do we not have logging set up at this point yet?

Reply:
Oops, knew I left debugging in there somewhere.